### PR TITLE
ioter log is written to a file

### PR DESCRIPTION
**Pull request by spdkimo**
Logs are printed on stdout are written to a file as well.
The format of file name is tmp/ioter_%y%m%d-%H%M.log

### CODE REVIEW

File: src/common/log.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from datetime import datetime
+
+class Log():
+    file = None
+
+    ## Init class ##
+    def __init__(self, path, name):
+        try:
+            Log.file = open(path+name+"_"+datetime.now().strftime("%y%m%d-%H%M")+".log", "w")
+        except Exception as e:
```
Comment (junyong-sim) [inline src/common/log.py:44]: The name of device logs is like `"230620-1049-lightbulb-0.log"`.
The naming rule for device logs is following.
 `${TIMESTAMP}"-"${DEVICE_ID_STR}"-"${DEVICE_NUM}".log`

So, how about we save it like following in order to sort easily ?
`tmp/%y%m%d-%H%M-ioter.log`

**Comment by jinwon0325kim** [comment]
Is it possible to save python error information in the log.

**Comment by junyong-sim** [comment]
> Is it possible to save python error information in the log.
link : https://stackoverflow.com/questions/38776104/redirect-stdout-and-stderr-to-same-file-using-python
When I tried to test it, stderr was not printed on terminal. But, otherwise it seems everything is fine.
```
+import sys
 
 class Log():
     file = None
@@ -41,6 +42,7 @@ class Log():
     def __init__(self, path, name):
         try:
             Log.file = open(path+name+"_"+datetime.now().strftime("%y%m%d-%H%M")+".log", "w")
+            sys.stderr = Log.file
         except Exception as e:
```

**Comment by spdkimo** [comment]
> Is it possible to save python error information in the log.

stderr is saved to the log file as well now

**Review by junyong-sim — APPROVED** [review]
LGTM

**Pull request merged**